src/routes/auth.route.ts

```diff
@@ -59,9 +76,33 @@ class AuthRouter extends BaseRoute {
     }
   }
 
+  async refreshRoute(
+    req: express.Request,
+    res: express.Response) {
+    const { refreshToken } = req.body;
+    const accessToken = req?.headers?.authorization?.split(' ')[1];
+    if (refreshToken && accessToken && req.user) {
+      const expressUser: ExpressUser = req.user;
+      const user: IUser = {
+        email: expressUser.email,
+        firstname: expressUser.firstname,
+        lastname: expressUser.lastname,
+        password: expressUser.password
+      }
```

Comment:
Nit: can be shortened to something like
```suggestion
      const {email, firstname, lastname, password } = req.user;
      const user: IUser = {
        email,
        firstname,
        lastname,
        password
      }
```

Reply:
true. fixed